src/Containers/CheckoutPage/CheckoutPage.js

```diff
@@ -0,0 +1,21 @@
+import classes from './CheckoutPage.module.css';
+import {CheckoutContext} from '../../store/CheckoutContext'
+import { useContext } from 'react';
+import AddressPage from './AddressPage/AddressPage'
+import PaymentPage from './PaymentPage/PaymentPage'
+import OrderPlacedModal from './OrderPlacedModal/OrderPlacedModal'
+import OrderSummary from './OrderSummary/OrderSummary';
```

Comment:
Hi Muthu,
I am assuming the path is like this ...:AddressPage(folder)>AddressPage.js(file)
A very minor suggestion here: you can create an index .js in the addressspage folder and club ur exports there.
So while importing it will look like this : import AddressPage from "./AddressPage"  (Just a little cleaner).

Please let me know if i am mistaken anywhere!!